tests/integration/kubernetes/k8s-confidential.bats

```diff
@@ -46,7 +51,9 @@ setup() {
 }
 
 teardown() {
-	[[ " ${SUPPORTED_HYPERVISORS[*]} " =~  " ${KATA_HYPERVISOR} " ]] ||  skip "Test not supported for ${KATA_HYPERVISOR}."
+	if ! [[ "${SUPPORTED_TEE_HYPERVISORS[@]}" =~ "${KATA_HYPERVISOR}" ]] && [[ "${SUPPORTED_NON_TEE_HYPERVISORS}" =~ "${KATA_HYPERVISOR}" ]]; then
```

Comment:
```suggestion
	if ! [[ "${SUPPORTED_TEE_HYPERVISORS[@]}" =~ "${KATA_HYPERVISOR}" ]] && ! [[ "${SUPPORTED_NON_TEE_HYPERVISORS}" =~ "${KATA_HYPERVISOR}" ]]; then
```

Otherwise the logic is not correct.